src/app/zap-templates/zcl/data-model/chip/keypad-input-cluster.xml

```diff
@@ -26,26 +26,26 @@ limitations under the License.
     <!-- TODO: Add feature map once it is available -->
     <description>This cluster provides an interface for controlling a device like a TV using action commands such as UP, DOWN, and SELECT.</description>
 
-    <command source="client" code="0x00" name="SendKey" optional="false">
+    <command source="client" code="0x00" name="SendKeyRequest" response="SendKeyResponse" optional="false">
       <description>Upon receipt, this SHALL process a keycode as input to the media device.</description>
-      <arg name="keyCode" type="KeypadInputCecKeyCode"/>
+      <arg name="keyCode" type="CecKeyCode"/>
```

Comment:
This is not the arg type in the spec.  It's `uint8` in the spec, no?

Reply:
Yes. As mentioned below I've added the Cec Key Codes. They are already predefined and client as well as server can know in advance what they are dealing with. They don't have to guess or search every time what's the CEC code list.

Reply:
I can file a ticket for spec update?